knip.jsonc

```
@@ -2,5 +2,5 @@
 	"$schema": "https://unpkg.com/knip@latest/schema.json",
 	"entry": ["src/index.ts!", "src/cli.ts!"],
 	"ignoreExportsUsedInFile": { "interface": true, "type": true },
-	"project": ["src/**/*.ts!"]
+	"project": ["src/**/*.ts!"],
```

Unrelated change - I'm guessing some tooling issue automatically did it? That would be something broken to fix outside of this PR.

```suggestion
	"project": ["src/**/*.ts!"]
```

I fixed this just now in #592 -> #593, so if you pull in the latest `main` branch commits the automatic comma thing should go away.

---

Yep, this change came about when I ran prettier (pnpm format --write). I did pull in main as you recommended, ran the prettier command again, and the automatic comma went away 🙌